### ISSUE TITLE
Cookies proxying problem

### ISSUE THREAD

**Issue by banjankri** [BrowserSync/browser-sync#467]
Hi,
I have an issue when using proxy mode of browser-sync and sending se-cookie response headers. For some reason the cookie value that is being set (security related) has outer quotes missing.

The value sent from server: "Lw3lzdJnMZtcoxEcjNaRNq0QeMC0l8amty20Nraz+5xrycCuijrhXHE881KK6Tn3uOCRjp4DhUjicRC2PnulIEXUJQMlaZ2So8wbGwIDBC2p4N0x4xM3Wp7vJoK9s84LFb3FKVVup5I="

Cookie value set in the browser: Lw3lzdJnMZtcoxEcjNaRNq0QeMC0l8amty20Nraz+5xrycCuijrhXHE881KK6Tn3uOCRjp4DhUjicRC2PnulIEXUJQMlaZ2So8wbGwIDBC2p4N0x4xM3Wp7vJoK9s84LFb3FKVVup5I= (outer double quotes missing)

When trying it out without proxy in between works like a charm.

Thanks,
Artur

**Comment by stipsan** [comment]
This is likely because browser-sync is using foxy for the proxy. Foxy got a option called cookies.stripDomain enabled by default.
This uses a cookie parsing utility that strips out the quotes, here's the code doing it: https://github.com/jshttp/cookie/blob/master/index.js#L54

I just submitted a pull request (#474) that enables you to pass this option in the proxy config object:
```js
    proxy: {
        target: 'localhost.dev',
        cookies: {stripDomain: false}
    }
```

Please note that the stripDomain funksjon is there to solve a problem with logins and if disabling it breaks your login functionality I don't know how to fix that as the cookie.parse utility will strip your quotes no matter what.
Here's how Foxy is parsing the cookie while stripping the domain: https://github.com/ShakyShane/foxy/blob/develop/lib/utils.js#L36

Hope this helps,
Stian

**Comment by shakyShane** [comment]
Fixed in 2.6.0
